superset/db_engine_specs/bigquery.py

```diff
@@ -78,6 +78,11 @@ class BigQueryParametersSchema(Schema):
     query = fields.Dict(required=False)
 
 
+class ErrorMessageType(TypedDict):
```

Comment:
Thinking about this object, I think there are 3, maybe 4 things:
1. error_category:str ("BigQuery Error", or "Database Error", "Missing Column" or "Unexpected Error")
2. error_message:str ("Invalid expression, the table 'users' does not include a 'gender' column ")
3. error_details[str:optional] (the expandable section)
4. error_code[str:optional] (3332), error from databases or in other context can have a unique code. This is specialized and may not be used most of the times